.env.sample

```diff
@@ -2,6 +2,8 @@ SESSION_SECRET=
 OUTGOING_TOKEN=
 INCOMING_HOOK_URL=https://hooks.slack.com/services/
 API_TOKEN=xoxp-
+BLOB_URL=https://jsonblob.com/api/jsonBlob/
```

Comment:
bad name for a config variable. RESERVED_NICK_URL

Also provide a sample to show how the json is formatted.